src/components/FormFields/index.js

```diff
@@ -104,39 +108,50 @@ class FormFields extends React.Component {
         let startTime
         let endTime
         let subEventKeys = Object.keys(this.props.editor.values.sub_events)
-        let key = subEventKeys.length > 0 ? Math.max.apply(null, subEventKeys)+1 : 1
+        let key = subEventKeys.length > 0 ? Math.max.apply(null, subEventKeys)+1 : 0
```

Comment:
Are you sure, that `this.props.editor.values.sub_event` will not be null or undefined (in this case, you will receive error)